mod/profperm.php

```diff
@@ -109,9 +109,9 @@ function profperm_content(App &$a) {
 	}
 
 	$o .= '<div id="prof-update-wrapper">';
-	if($change) 
+	if($change)
```

Comment:
It does indeed, especially since the blank line after still had a tab as if someone removed the closing brace but not the tab. Good job Sherlock! 🕵️